### PR TITLE
Beacd 46 fix: fixed issue with conflicts

### PR DESCRIPTION
**Pull request by ryanniven**

### CODE REVIEW

File: app/utils/NextPage.scala [hunk 1]
```diff
@@ -35,18 +35,28 @@ import models.other.oil.SelectAnOilService.{RebatedOilsEnquiryService, TiedOilsE
...
 trait NextPage[A, B] {
-  def get(b: B)(implicit urlHelper: UrlHelper): Call
+  def get(b: B)(implicit urlHelper: UrlHelper, request: Request[_]): Call
 }
 
 object NextPage {
 
-  private val GET: String = "GET"
+  implicit val whichPensionSchemeToAdd: NextPage[WhichPensionSchemeToAddId.type,
+    models.employer.pension.WhichPensionSchemeToAdd] = {
```
Comment (kurtlogan) [inline app/utils/NextPage.scala]: Consider not fully qualifying model

File: app/utils/NextPage.scala [hunk 2]
```diff
@@ -55,8 +65,8 @@ object NextPage {
   }
 
   implicit val doYouWantToAddImportExport: NextPage[DoYouWantToAddImportExportId.type, DoYouWantToAddImportExport] = {
-    new NextPage[DoYouWantToAddImportExportId.type, DoYouWantToAddImportExport] {
-      override def get(b: DoYouWantToAddImportExport)(implicit urlHelper: UrlHelper): Call =
+    new NextPage[DoYouWantToAddImportExportId.type, models.other.importexports.DoYouWantToAddImportExport] {
```
Comment (kurtlogan) [inline app/utils/NextPage.scala]: Consider not fully qualifying model

File: app/utils/UrlHelper.scala [hunk 3]
```diff
@@ -46,11 +49,19 @@ class UrlHelper @Inject()(val appConfig: FrontendAppConfig) {
...
   }
+
+  def getPortalURL(key: String)(implicit request: Request[_]): String = {
+    val portalUrl = appConfig.getPortalUrl(key)
+    appendLanguage(portalUrl)(request)
+  }
+
+  private def appendLanguage(url: String)(implicit request: Request[_]) = {
+    val lang = if (LanguageUtils.getCurrentLang == LanguageUtils.Welsh) "lang=cym" else "lang=eng"
+    val token = if (url.contains("?")) "&" else "?"
```
Comment (kurtlogan) [inline app/utils/UrlHelper.scala]: What if the url ends with a `?`, this would remove lang and create a nameless parameter

Comment (prashantchoudhari) [reply app/utils/UrlHelper.scala]: In our case - URL's are written by us currently not generated, so I am not hoping this to happen. we can sit together to try this by writing test.

**Comment by prashantchoudhari** [comment]
@kurtlogan - FYA

**Pull request merged**